packages/core/src/app/locale/translations/en.json

```diff
@@ -461,6 +461,9 @@
             "label": "Yes, I agree with the <a href=\"{url}\" target=\"_blank\">privacy policy</a>.",
             "heading": "Privacy Policy"
         },
+        "tax": {
```

Comment:
🍻 I note we have `cart.taxes_text` already - would it make sense to have this present there also?

e.g. `cart.taxes_inclusive_text`?

...or are you perhaps thinking that will move to `exclusive_label` here?

Reply:
I was thinking we can have a separate section of the `inclusive_text`, maybe makes sense to move it as `cart.taxes_inclusive_text` if that's the only bit of text for that section 🤔